webapp/channels/src/selectors/posts.ts

```diff
@@ -51,3 +59,69 @@ export function isInlineImageVisible(state: GlobalState, postId: string, imageKe
 
     return getGlobalItem(state, StoragePrefixes.INLINE_IMAGE_VISIBLE + currentUserId + '_' + postId + '_' + imageKey, !imageCollapsed);
 }
+
+export function makeCanWrangler() {
+    return createSelector(
+        'makeCanWrangler',
+        getConfig,
+        getCurrentUser,
+        moveThreadsEnabled,
+        (_state: GlobalState, channelType: Channel['type']) => channelType,
+        (_state: GlobalState, _channelType: Channel['type'], replyCount: number) => replyCount,
+        (config: Partial<ClientConfig>, user: UserProfile, enabled: boolean, channelType: Channel['type'], replyCount: number) => {
+            if (!enabled) {
+                return false;
+            }
+            const {
+                WranglerPermittedWranglerUsers,
+                WranglerAllowedEmailDomain,
+                WranglerMoveThreadMaxCount,
+                WranglerMoveThreadFromPrivateChannelEnable,
+                WranglerMoveThreadFromDirectMessageChannelEnable,
+                WranglerMoveThreadFromGroupMessageChannelEnable,
+            } = config;
+
+            let permittedUsers: string[] = [];
+            if (WranglerPermittedWranglerUsers && WranglerPermittedWranglerUsers !== '') {
+                permittedUsers = WranglerPermittedWranglerUsers?.split(',');
+            }
+
+            let allowedEmailDomains: string[] = [];
+            if (WranglerAllowedEmailDomain && WranglerAllowedEmailDomain !== '') {
+                allowedEmailDomains = WranglerAllowedEmailDomain?.split(',') || [];
+            }
+
+            if (permittedUsers?.length > 0) {
```

Comment:
permittedUsers seems to be always defined.
```suggestion
            if (permittedUsers.length > 0) {
```